### PR TITLE
Panel submit

### PR DESCRIPTION
**Pull request by adriana-alter**
Resolves https://jira.nypl.org/browse/SIMPLY-1882

### CODE REVIEW

File: __tests__/components/Panel-test.tsx [hunk 1]
```diff
@@ -110,4 +112,22 @@ describe("Panel", () => {
...
   });
 
+  it("should optionally accept a callback", () => {
+    let spyToggle = Sinon.spy(wrapper.instance(), "toggle");
+    let spyOverride = Sinon.spy(wrapper.instance(), "overrideEnter");
+    let onEnter = Sinon.stub();
+    let input = <input />;
+    wrapper.setProps({
+      toggle: spyToggle,
+      overrideEnter: spyOverride,
```
Comment (EdwinGuzman) [inline __tests__/components/Panel-test.tsx]: `overrideEnter` and `toggle` are not actually a props on `Panel` but methods. This doesn't seem like the right way to test this.

Comment (EdwinGuzman) [reply __tests__/components/Panel-test.tsx]: You're spying them above, so no need to set them as props.

**Comment by EdwinGuzman** [comment]
This should somehow be tested with Willa before moving on to make sure it does meet accessibility standards.

**Comment by adriana-alter** [comment]
As I see it, this is really a bug fix; circ manager forms containing panels used to submit on enter, and then I accidentally broke that functionality when I imported the reusable Panel.  I would love to check this (and, if possible, the rest of the reusable components!) with Willa when she gets back from vacation...but in the meantime, I'd really like to get this merged, since all it does is restore a previously-implemented behavior.  There may well be a more accessible way to do this...but the bug is making it so that, right now, the only way to submit forms is by clicking the button, which definitely does NOT meet accessibility standards!

**Comment by EdwinGuzman** [comment]
Alright, we need to check more thoroughly to not break previous functionality.

**Pull request merged**